features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | 999993045           | 0263          | Surinaamse                 |                | 301         | Vaststelling bezit vreemde nationaliteit        | 00000000              |
+      | 999993367           | 0000          | Onbekend                   |                | 311         | Vaststelling onbekende nationaliteit            | 00000000              |
+      | 000009866           |               |                            | B              | 310         | Vaststelling bijzonder Nederlanderschap         | 19570115              |
+      | 999994748           |               |                            | V              | 310         | Vaststelling bijzonder Nederlanderschap         | 19750615              |                                                                                                     |
+
+  Rule: wanneer een veld van een datum wordt gevraagd, wordt de datum geleverd
+
+    @gba
+    Abstract Scenario: met fields is gevraagd om <fields>
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999993653 |
+      | <GBA veld>          | 20131102  |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993653                       |
+      | fields              | <veld>.<datumveld>              |
+      Dan heeft de response een persoon met alleen de volgende gegevens
+      | naam   | waarde   |
+      | <veld> | 20131102 |
+
+      Voorbeelden:
+      | datumveld   | GBA veld                                  | veld                        |
+      | type        | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | langFormaat | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | datum       | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | jaar        | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | maand       | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | onbekend    | datum eerste inschrijving GBA/RNI (68.10) | datumEersteInschrijvingGBA  |
+      | type        | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+      | langFormaat | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+      | datum       | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+      | jaar        | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+      | maand       | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+      | onbekend    | datum inschrijving (09.20)                | datumInschrijvingInGemeente |
+
+    @gba
+    Abstract Scenario: met fields is gevraagd om <fields>
```

Comment:
Mij lijkt dat dit scenario geen relatie heeft met de in regel 563 beschreven rule.

Reply:
Ik heb ook het idee dat de situatie intussen gewijzigd is.